include/envoy/event/dispatcher.h

```diff
@@ -38,6 +38,16 @@ class Dispatcher {
    */
   virtual void clearDeferredDeleteList() PURE;
 
+  /**
+   * Create a server connection.
+   * @param accept_socket supplies a socket with an open file descriptor and connection metadata
+   *        to use for the connection. Takes ownership of the accept_socket.
+   * @param ssl_ctx supplies the SSL context to use, if not nullptr.
+   * @return Network::ConnectionPtr a client connection that is owned by the caller.
+   */
+  virtual Network::ConnectionPtr createConnection(Network::AcceptSocketPtr&& accept_socket,
```

Comment:
I would probably call this `createServerConnection` to match `createClientConnection`

Reply:
Will change.

Reply:
There is no ServerConnection class, ClientConnection inherits from Connection, so from that viewpoint createConnection() is correct. Hosts also have a createConnection method, so createServerConnection makes grepping the source easier.